hack/update-generated-completions.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STI_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+source "${STI_ROOT}/hack/common.sh"
+
+sti::build::build_binaries "$@"
+
+echo "+++ Updating Bash completion (${STI_ROOT}/contrib/bash/s2i)"
```

Comment:
The `${STI_ROOT}` part in the message will look weird, something like `/path/to/s2i/hack/../contrib/bash/s2i`.
Not very useful, we could just drop that part and keep only the relative path `contrib/bash/s2i`, or even drop it altogether. WDYT?